app/components/assets/asset-alert-cards.tsx

```diff
@@ -0,0 +1,82 @@
+import { useLoaderData } from "@remix-run/react";
+import { EllipsisIcon } from "lucide-react";
+import { type loader } from "~/routes/_layout+/assets.$assetId.overview";
+import { tw } from "~/utils/tw";
+import { resolveTeamMemberName } from "~/utils/user";
+import { Button } from "../shared/button";
+import {
+  Tooltip,
+  TooltipContent,
+  TooltipProvider,
+  TooltipTrigger,
+} from "../shared/tooltip";
+import When from "../when/when";
+
+type AssetAlertCardsProps = {
+  className?: string;
+  style?: React.CSSProperties;
+};
+
+export default function AssetAlertCards({
+  className,
+  style,
+}: AssetAlertCardsProps) {
+  const { alerts } = useLoaderData<typeof loader>();
+
+  if (!alerts.length) {
+    return;
+  }
+
+  return (
+    <div className={tw("rounded border bg-white", className)} style={style}>
+      <div className="flex items-center justify-between gap-4 border-b px-4 py-3">
+        <h5>Reminders</h5>
+
+        <Button variant="block-link-gray" className="text-gray-400">
+          <EllipsisIcon />
```

Comment:
We dont have any actions to put here so lets just replace this with a link saying "View all"